scilpy/io/hdf5.py

```diff
@@ -106,7 +104,9 @@ def reconstruct_sft_from_hdf5(hdf5_handle, group_keys, space=Space.VOX,
             for sub_key in hdf5_handle[group_key].keys():
                 if sub_key not in ['data', 'offsets', 'lengths']:
                     data = hdf5_handle[group_key][sub_key]
-                    if data.shape == hdf5_handle[group_key]['offsets'].shape:
+                    if data.shape == hdf5_handle[group_key]['offsets'].shape or np.isreal(data):
+                        if data.shape == ():
```

Comment:
Maybe an example? A comment to say that this is ''data_per_group'' a single value to represent the whole connection and then that you fake extend it so it can be stored in DPS?